.github/workflows/testing-scheduled.yml

```diff
@@ -0,0 +1,30 @@
+name: new dependency test (scheduled)
+
+on:
+  schedule:
+    - cron: '30 5 * * 1' 
+
```

Comment:
I don't use cron much. I am also not sure by reading 
https://crontab.guru/

Is it once per month or weekly? I would prefer weekly.

Reply:
https://crontab.guru/every-week
weekly
```0 0 * * 0```